### PR TITLE
Standalone edit respondent details

### PR DESCRIPTION
**Pull request by AdeHarrison**
# Motivation and Context
Converted edit_respondent_details.feature to standalone

#Link
[Trello](https://trello.com/c/XExcWx0d/334-acceptance-tests-convert-editrespondentdetails-to-standalone)

### CODE REVIEW

**Comment by AdeHarrison** [comment]
[Acceptance Tests: convert edit_respondent_details to standalone](https://trello.com/c/XExcWx0d/334-acceptance-tests-convert-editrespondentdetails-to-standalone)

**Comment by jcox-dev** [comment]
Doesn't look to be any merge conflicts 👍

**Comment by AdeHarrison** [comment]
@jcox-dev added a fix in for @us058-s003 failure, can you re-check please

File: controllers/database_controller.py [hunk 1]
```diff
@@ -123,3 +123,17 @@ def unenrol_respondent_in_survey(survey_id):
 
     sql_statement_delete_enrolment = f"delete from partysvc.enrolment where survey_id = '{survey_id}'"
     execute_sql(sql_string=sql_statement_delete_enrolment, database_uri=Config.PARTY_DATABASE_URI)
+
+
+def get_different_respondent_email_address(respondent_email_address):
+    sql_statement = "select p.email_address " \
+                    "from partysvc.respondent p " \
+                    f"where email_address <> '{respondent_email_address}' " \
```
Comment (NickGrantONS) [inline controllers/database_controller.py]: this should be `p.email_address`

**Pull request merged**